src/docfx/build/legacy/metadata/LegacyMetadata.cs

```diff
@@ -80,7 +86,10 @@ public static JObject GenerateLegacyRedirectionRawMetadata(Docset docset, PageMo
 
             rawMetadata["_op_canonicalUrlPrefix"] = $"{docset.Config.BaseUrl}/{docset.Locale}/{docset.Config.DocumentId.SiteBasePath}/";
 
-            rawMetadata["monikers"] = new JArray(pageModel.Monikers);
+            if (pageModel.Monikers.Count > 0)
```

Comment:
When monikers is empty, is `group` still in the output?

Reply:
nope, removed in the previous PR